src/stars.css

```diff
@@ -0,0 +1,26 @@
+.card__rating-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+  margin-bottom: 24px;
+}
+  
+.stars {
+  display: flex;
+  line-height: 16px;
+}
+  
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+  
+.stars--4 > .stars__star:nth-child(-n + 4) {
```

Comment:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.